README.md

```diff
@@ -28,7 +29,7 @@ structures:
 * [Bifunctor](#bifunctor)
 * [Profunctor](#profunctor)
 
-<img src="figures/dependencies.png" width="888" height="340" />
+<img src="figures/dependencies.png" width="1068" height="347" />
```

Comment:
Rather than the image's natural dimensions, I think we want the dimensions of the scaled image on GitHub:

```html
<img src="figures/dependencies.png" width="888" height="289" />
```